ims/get_admin_organizations.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+
+package ims
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// GetAdminOrganizationsRequest is the request for GetOrganizations.
+type GetAdminOrganizationsRequest struct {
+	Guid         string
+	AuthSrc      string
+	ServiceToken string
+	ApiVersion   string
+	ClientID     string
+}
+
+// GetAdminOrganizationsResponse is the response for GetOrganizations.
+type GetAdminOrganizationsResponse struct {
+	Response
+}
+
+// GetAdminOrganizationsWithContext reads the user organizations associated to a
+// given access token. It returns a non-nil response on success or an error on
+// failure.
+func (c *Client) GetAdminOrganizationsWithContext(ctx context.Context, r *GetAdminOrganizationsRequest) (*GetAdminOrganizationsResponse, error) {
+	if r.ApiVersion == "" {
```

Comment:
does it make sense to expose the API version? i assume every version will have a different payload, hence, a different version would need to be supported by the method.

Reply:
I think it does.

The changes between versions are generally small and transparent to the purpose of the library. Changes like adding a new value or a new field don't need new support and it is practical for the clients to update their API version used easily as they need.